src/coffee/user_agent.coffee

```diff
@@ -16,6 +16,7 @@ UAs =
   ios: /(ipad|iphone|ipod)/i
   unix: /(linux|openbsd|freebsd|netbsd)/i
   osx_catalina: /macintosh.+10(_|\.)15/i
+  osx_big_sur: /macintosh.+((10(_|\.)16)|(11(_|\.)))/i
```

Comment:
Interesting, it uses both 10.16 and 11.0 as its version number?

Reply:
Yeah: https://www.reddit.com/r/MacOSBeta/comments/htemp7/what_are_chrome_safari_useragent_in_big_sur/